<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortSoftWareClass</class>
 <widget class="QMainWindow" name="SerialPortSoftWareClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>983</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPortSoftWare</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>321</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>电机控制</string>
    </property>
    <widget class="QPushButton" name="downBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton:enabled {
        background: rgb(120, 170, 220);
        color: white;
    }

    QPushButton:!enabled {
        background: rgb(180, 180, 180);
        color: white;
    }

    QPushButton:enabled:hover {
        background: rgb(100, 160, 220);
    }

    QPushButton:enabled:pressed {
        background: rgb(0, 78, 161);
    }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="upBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>93</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton:enabled {
        background: rgb(120, 170, 220);
        color: white;
    }

    QPushButton:!enabled {
        background: rgb(180, 180, 180);
        color: white;
    }

    QPushButton:enabled:hover {
        background: rgb(100, 160, 220);
    }

    QPushButton:enabled:pressed {
        background: rgb(0, 78, 161);
    }
  </string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="rightBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>93</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton:enabled {
        background: rgb(120, 170, 220);
        color: white;
    }

    QPushButton:!enabled {
        background: rgb(180, 180, 180);
        color: white;
    }

    QPushButton:enabled:hover {
        background: rgb(100, 160, 220);
    }

    QPushButton:enabled:pressed {
        background: rgb(0, 78, 161);
    }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="leftBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>93</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton:enabled {
        background: rgb(120, 170, 220);
        color: white;
    }

    QPushButton:!enabled {
        background: rgb(180, 180, 180);
        color: white;
    }

    QPushButton:enabled:hover {
        background: rgb(100, 160, 220);
    }

    QPushButton:enabled:pressed {
        background: rgb(0, 78, 161);
    }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="stopBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>93</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton:enabled {
        background: rgb(120, 170, 220);
        color: white;
    }

    QPushButton:!enabled {
        background: rgb(180, 180, 180);
        color: white;
    }

    QPushButton:enabled:hover {
        background: rgb(100, 160, 220);
    }

    QPushButton:enabled:pressed {
        background: rgb(0, 78, 161);
    }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="rsetBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>320</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton:enabled {
        background: rgb(120, 170, 220);
        color: white;
    }

    QPushButton:!enabled {
        background: rgb(180, 180, 180);
        color: white;
    }

    QPushButton:enabled:hover {
        background: rgb(100, 160, 220);
    }

    QPushButton:enabled:pressed {
        background: rgb(0, 78, 161);
    }</string>
     </property>
     <property name="text">
      <string>复位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>状态：</string>
     </property>
    </widget>
    <widget class="QLabel" name="motorLab">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>441</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>接收区</string>
    </property>
    <widget class="QTextEdit" name="receiveEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>发送区</string>
    </property>
    <widget class="QTextEdit" name="sendEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>651</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>视频</string>
    </property>
    <widget class="QLabel" name="videoLab">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>651</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>551</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>传感器数据</string>
    </property>
    <widget class="QLabel" name="serialStatusLab">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>491</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="typeLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>类型</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="numLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>数值</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="statusLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>等级</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="tempLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>温度：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="tempNumLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="tempStatusLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="humLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>湿度：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="humNumLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="humStatusLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="combusLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>可燃气体：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="combusNumLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="combusTatausLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="chLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>甲醛：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="oxyNumLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="oxyStatusLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="pmLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>PM2.5:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="pmNumLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="pmStatusLab">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>470</y>
      <width>221</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>网络设置</string>
    </property>
    <widget class="QLabel" name="portLab">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>72</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1200</string>
     </property>
    </widget>
    <widget class="QLabel" name="netStatusLab">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>72</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="portLab_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>本地端口：</string>
     </property>
    </widget>
    <widget class="QLabel" name="connectstatusLab">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>连接状态：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="imageStatusEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>201</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>211</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>173</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblPortName">
          <property name="text">
           <string>串口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbPortName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblBaudRte">
          <property name="text">
           <string>波特率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbBaudRate"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblDataBits">
          <property name="text">
           <string>数据位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbDataBits"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblParity">
          <property name="text">
           <string>奇偶校验位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbParity"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lblStopBits">
          <property name="text">
           <string>停止位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbStopBits"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblFlowControl">
          <property name="text">
           <string>流控制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbFlowControl"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="OpenBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton:enabled {
        background: rgb(120, 170, 220);
        color: white;
    }

    QPushButton:!enabled {
        background: rgb(180, 180, 180);
        color: white;
    }

    QPushButton:enabled:hover {
        background: rgb(100, 160, 220);
    }

    QPushButton:enabled:pressed {
        background: rgb(0, 78, 161);
    }</string>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>groupBox_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="serial_action"/>
   <addaction name="setting_action"/>
   <addaction name="camera_action"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="serial_action">
   <property name="icon">
    <iconset resource="SerialPortSoftWare.qrc">
     <normaloff>:/SerialPortSoftWare/Resources/usb.ico</normaloff>:/SerialPortSoftWare/Resources/usb.ico</iconset>
   </property>
   <property name="text">
    <string>串口设置</string>
   </property>
  </action>
  <action name="setting_action">
   <property name="icon">
    <iconset resource="SerialPortSoftWare.qrc">
     <normaloff>:/SerialPortSoftWare/Resources/command.ico</normaloff>:/SerialPortSoftWare/Resources/command.ico</iconset>
   </property>
   <property name="text">
    <string>指令设置</string>
   </property>
   <property name="toolTip">
    <string>指令设置</string>
   </property>
  </action>
  <action name="camera_action">
   <property name="icon">
    <iconset resource="SerialPortSoftWare.qrc">
     <normaloff>:/SerialPortSoftWare/Resources/camera.ico</normaloff>:/SerialPortSoftWare/Resources/camera.ico</iconset>
   </property>
   <property name="text">
    <string>接收视频</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SerialPortSoftWare.qrc"/>
 </resources>
 <connections/>
</ui>
